common/effect_localization/wc_magic_effects.txt

```diff
@@ -0,0 +1,28 @@
+ADD_SOUL = {
+	global = ADD_SOUL
+	first = ADD_SOUL
+	third = ADD_SOUL
+}
+ADD_SOULS = {
+	global = ADD_SOULS
+	first = ADD_SOULS
+	third = ADD_SOULS
+}
+REMOVE_SOUL = {
+	global = REMOVE_SOUL
+	first = REMOVE_SOUL
+	third = REMOVE_SOUL
+	
+	global_past = REMOVE_SOUL
+	first_past = REMOVE_SOUL
+	third_past = REMOVE_SOUL
+}
+REMOVE_SOULS = {
+	global = REMOVE_SOULS
+	first = REMOVE_SOULS
+	third = REMOVE_SOULS
+	
+	global_past = REMOVE_SOULS
+	first_past = REMOVE_SOULS
+	third_past = REMOVE_SOULS
+}
```

Comment:
You don't need both remove and add effects.
Check `change_dark_frenzy_effect`.

Reply:
Fixed in 
4a7ec34